app/serializers/base_serializer.rb

```diff
@@ -123,6 +122,18 @@ def show_policy?
     return respond_to_policy && respond_to_manage && can_manage
   end
 
+  def determine_submitter(object)
+    # FIXME: needs to be the creators for assets
```

Comment:
this comment is redundant and can be removed

Reply:
Comment removed